core/DataAccess/ArchiveSelector.php

```diff
@@ -92,10 +92,14 @@ public static function getArchiveIdAndVisits(ArchiveProcessor\Parameters $params
             return [false, $visits, $visitsConverted, true, $tsArchived];
         }
 
+        if (!empty($minDatetimeArchiveProcessedUTC) && (!is_object($minDatetimeArchiveProcessedUTC) || !$minDatetimeArchiveProcessedUTC instanceof Date)) {
```

Comment:
If the var isn't an object then it will never be a `Date`, right?

Reply:
Could remove it. Guess in case it was some different class but that wouldn't work anyway